app/models/language_skill.rb

```diff
@@ -1,6 +1,12 @@
 class LanguageSkill < ApplicationRecord
   belongs_to :languageable, polymorphic: true, optional: true
 
+  scope :native_languages, lambda {
+    where(level: 'native_speaker')
+      .order("CASE WHEN language = 'DE' THEN 1 WHEN language != 'DE' THEN 2 END")
```

Comment:
`CASE WHEN language = 'DE' THEN 1 ELSE 2 END`

Reply:
Thx. That looks much better.. 👍 😄